request-body-canonicalization/latest/index.md

```diff
@@ -0,0 +1,279 @@
+# Request Body Canonicalization
+
+## Abstract
+
+Originally, CDX files were only used to index web archives containing GET requests. As browser-based capture methods can record non-GET requests such as those generated by JavaScript, a way for CDX/CDXJ index records to differentiate based on request method and request body is needed. This document describes the mechanism used for encoding the request method and body in the CDX/CDXJ key by appending additional query parameters, as originally implemented by pywb.
+
+## Conformance
+
+As well as sections marked as non-normative, all authoring guidelines, diagrams, examples, and notes in this specification are non-normative. Everything else in this specification is normative.
+
+The key words MAY and MUST in this document are to be interpreted as described in BCP 14 [RFC2119][1] [RFC8174][2] when, and only when, they appear in all capitals, as shown here.
+
+## Terminology
+
+- CDX
+- CDXJ
+- WACZ
+- WARC
+
+## Introduction
+
+### Web Archive Formats (WARC and WACZ)
+
+Web archiving data is often stored in specialized formats, which include a full record of the HTTP network traffic as well as additional metadata. The archived data is often accessed via random-access, loading the appropriate chunks of data based on URLs requested by end users.
+
+Web archiving data is often stored in two key file formats:
+
+1. WARC — A widely accepted [ISO standard][3] used by many institutions around the world for storing web archive data.
+2. WACZ — A new format [developed by Webrecorder][4] for packaging WARCs with other web archive data which supports random-access reads.
```

Comment:
Would further clarify that the extra metadata is what makes the random access reads possible / performant
```suggestion
2. WACZ — A new format [developed by Webrecorder][4] for packaging WARCs with other web archive data enabling efficient random-access reads.
```